next-docs/tailwind.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
         developer: 'rgba(149, 241, 213, 0.9)',
         transparent: 'transparent',
         current: 'currentColor',
+        backdrop: 'rgba(0, 0, 0, 0.56)'
```

Comment:
What is the reason to replace backdrop color in next-docs/tailwind.config.js as it was added in moon-preset ?

Reply:
removed